### ISSUE TITLE
Warnings in Parcel

### ISSUE THREAD

**Issue by ghost** [hustcc/timeago.js#218]
I'm getting the following warnings when using timeago.js in a webapp build with Parcel:
![image](https://user-images.githubusercontent.com/49151515/75667055-3e0b1a00-5c77-11ea-832b-6a8d30d1182c.png)

**Comment by hustcc** [comment]
because of npm package do not contains src file?

**Comment by ghost** [comment]
@hustcc correct, but then there should be no references to the src folder at least.

**Comment by hustcc** [comment]
may be I should release a new version which contains `src`, or remove all the sourcemap files.

**Comment by ulken** [comment]
Still seeing this. Any updates?

**Comment by ulken** [comment]
@hustcc I could give it a try if you decide which route you want to go, i.e. what you want to be done.
